camelot/view/controls/delegates/floatdelegate.py

```diff
@@ -55,20 +58,33 @@ def get_editor_class(cls):
     @classmethod
     def get_standard_item(cls, locale, model_context):
         minimum, maximum = model_context.field_attributes.get('minimum'), model_context.field_attributes.get('maximum')
+        minimum = minimum if minimum is not None else constants.camelot_minfloat
+        maximum = maximum if maximum is not None else constants.camelot_maxfloat
         model_context.field_attributes.update({
-            'minimum': minimum if minimum is not None else constants.camelot_minfloat,
-            'maximum': maximum if maximum is not None else constants.camelot_maxfloat,
+            'minimum': minimum,
```

Comment:
ik vermoed dat die field_attributes update hier nt meer nodig is, die was er om vroeger zeker te zijn dat de field attributes die naar de editor gingen min/max hadden